vectordb_bench/backend/clients/pgvector/pgvector.py

```diff
@@ -113,24 +113,64 @@ def init(self) -> Generator[None, None, None]:
             self.conn.commit()
 
         index_param = self.case_config.index_param()
+        
+        reranking = self.case_config.search_param()["reranking"]
+        column_name = (
```

Comment:
@Sheharyar570 this part was a bit tricky to understand, just want to make sure you've reviewed the queries built with and without quantization.

Reply:
Yes I did review the queries with and without quantization. I believe this code could be refactored, especially the variable names. I'm thinking of renaming `column_name` to `field_name`?